torchvision/models/inception.py

```diff
@@ -69,7 +70,8 @@ def __init__(
         aux_logits: bool = True,
         transform_input: bool = False,
         inception_blocks: Optional[List[Callable[..., nn.Module]]] = None,
-        init_weights: Optional[bool] = None
+        init_weights: Optional[bool] = None,
+        random_state_numpy: np.random.RandomState = np.random.RandomState(),
```

Comment:
if we ever add a new parameter (I'm not sure we should for now, but we might have to), the default should be `None`

Reply:
Hum, we don't pass RNG in any of torchvision functions as of now, so I would rather not do it here, as it would involve a larger discussion.